### PR TITLE
Adds magnet support

### PR DESCRIPTION
**Pull request by PyR8zdl**
Adds magnet links to torrent page and search results.

Enable in config/torrent.php

### CODE REVIEW

File: resources/views/torrent/results.blade.php [hunk 1]
```diff
@@ -143,13 +143,29 @@ class="torrent-poster-img-small show-poster" alt="@lang('torrent.poster')">
                                     <i class="{{ config('other.font-awesome') }} fa-download"></i>
                                 </button>
                             </a>
+                            @if (config('torrent.magnet') == 1)
```
Comment (HDVinnie) [inline resources/views/torrent/results.blade.php]: Is this supposed to be here twice?

File: resources/views/torrent/results.blade.php [hunk 2]
```diff
@@ -143,13 +143,29 @@ class="torrent-poster-img-small show-poster" alt="@lang('torrent.poster')">
...
+                            </a>
+                            @endif
                         @else
                             <a href="{{ route('download', ['id' => $torrent->id]) }}">
                                 <button class="btn btn-primary btn-circle" type="button" data-toggle="tooltip"
                                     data-original-title="@lang('common.download')">
                                     <i class="{{ config('other.font-awesome') }} fa-download"></i>
                                 </button>
                             </a>
+                            @if (config('torrent.magnet') == 1)
```
Comment (HDVinnie) [inline resources/views/torrent/results.blade.php]: In relation to comment above.

File: resources/views/torrent/results.blade.php (hunk 1 above)
Comment (PyR8zdl) [reply resources/views/torrent/results.blade.php]: Yeah this was a silly mistake on my end. I'll clean it up. ;)

**Comment by masterfo78** [comment]
Don't Go no Upload with magnet

**Comment by HDVinnie** [comment]
> Don't Go no Upload with magnet

English?

**Comment by masterfo78** [comment]
When I use magnet it only shows the meta in the seedbox. No download is possible.

**Comment by torman365** [comment]
That is normal, downloads do work and show name will appear once completed

**Comment by masterfo78** [comment]
For me in the client, only a long number ends with .meta. The download does not start. In utorrent I have the name but the file does not start there either.

**Comment by torman365** [comment]
how are you adding the link to your client? you can right click the magnet link,copy  magnet link and then paste it into your clients download box where it says "torrent URL"

**Comment by masterfo78** [comment]
Yes the torrent url is from the tracker

**Comment by torman365** [comment]
yes but how are you adding it to your client?

**Comment by masterfo78** [comment]
per klick

**Comment by torman365** [comment]
try what i said, right click the magnet icon on the torrent page, then "copy link location" and then add that to your client in the "torrent URL" BOX

**Comment by masterfo78** [comment]
This also does not show with just the hash number

**Pull request merged**